src/components/MessageSection.js

```diff
@@ -24,7 +24,7 @@ function createMessageList (messages) {
       messageDate = new Date(m.created_at)
       diff = Math.abs(headerDate - messageDate)
       greaterThanMax = Math.floor(diff / 60000) > MAX_MINS_TO_BATCH
-      isHeader = greaterThanMax || m.user.id !== currentHeader.user.id
+      isHeader = greaterThanMax || m.user.id !== currentHeader.user.id || !m.created_at // last case means pending
```

Comment:
Yes, but otherwise we miss out on the 'sending...' feature... maybe that doesn't matter. but it will be kind of a bad UX if the comment creation fails, in this instances where its not a header